### PR TITLE
KOGITO-6507 Added example of Serverless Workflow consuming events over quarkus-http connector

### PR DESCRIPTION
**Pull request by hbelmiro**
JIRA: https://issues.redhat.com/browse/KOGITO-6507
Added example of Serverless Workflow consuming events over quarkus-http connector (SmallRye Reactive Messaging)

### CODE REVIEW

File: kogito-quarkus-examples/serverless-workflow-consuming-events-over-http-quarkus/README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Kogito Serverless Workflow - Consuming Events Over HTTP Example
+
+## Description
+
+This example contains a simple workflow service that consumes events over HTTP. 
+The service is described using JSON format as defined in the 
+[CNCF Serverless Workflow specification](https://github.com/serverlessworkflow/specification).
+This is a simple workflow that, after started, waits for an event to be published over HTTP.
+When the event is received, the workflow prints the event content to the console.
```
Comment (ricardozanini) [inline kogito-quarkus-examples/serverless-workflow-consuming-events-over-http-quarkus/README.md]:
```suggestion
This example contains a simple workflow service that consumes events over HTTP. 
The service is described using JSON format as defined in the 
[CNCF Serverless Workflow specification](https://github.com/serverlessworkflow/specification).
It's a simple workflow that, after starting, waits for an event to be published over HTTP.
Then the workflow prints the event content to the console when the event is received.
```

Comment (ricardozanini) [reply kogito-quarkus-examples/serverless-workflow-consuming-events-over-http-quarkus/README.md]: I think it's worth mentioning that this is the infrastructure required to integrate with Knative Eventing: https://knative.dev/docs/eventing/

> Knative Eventing uses standard HTTP POST requests to send and receive events between event producers and sinks. These events conform to the CloudEvents specifications, which enables creating, parsing, sending, and receiving events in any programming language.

File: kogito-quarkus-examples/serverless-workflow-consuming-events-over-http-quarkus/README.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+The endpoint for the broker is automatically generated by SmallRye.
```
Comment (ricardozanini) [inline kogito-quarkus-examples/serverless-workflow-consuming-events-over-http-quarkus/README.md]: I didn't get what you meant by this sentence.

**Review by ricardozanini — APPROVED** [review]
Pretty nice! Many thanks for this example. I just add some minor comments.

File: kogito-quarkus-examples/serverless-workflow-consuming-events-over-http-quarkus/README.md (hunk 2 above)
Comment (hbelmiro) [reply kogito-quarkus-examples/serverless-workflow-consuming-events-over-http-quarkus/README.md]: Actually this sentence is not needed. I'll remove it.

**Review by ricardozanini — APPROVED** [review]

**Review by fjtirado — APPROVED** [review]

**Pull request merged**